diff --git a/src/add-ons/translators/Jamfile b/src/add-ons/translators/Jamfile
index ab861a7..4cbb8c3 100644
--- a/src/add-ons/translators/Jamfile
+++ b/src/add-ons/translators/Jamfile
@@ -9,6 +9,8 @@ SubInclude HAIKU_TOP src add-ons translators ico ;
 SubInclude HAIKU_TOP src add-ons translators jpeg ;
 SubInclude HAIKU_TOP src add-ons translators jpeg2000 ;
 SubInclude HAIKU_TOP src add-ons translators pcx ;
+SubInclude HAIKU_TOP src add-ons translators people ;
+SubInclude HAIKU_TOP src add-ons translators vcard ;
 SubInclude HAIKU_TOP src add-ons translators png ;
 SubInclude HAIKU_TOP src add-ons translators ppm ;
 SubInclude HAIKU_TOP src add-ons translators raw ;
diff --git a/src/kits/Jamfile b/src/kits/Jamfile
index b635fda..870caec 100644
--- a/src/kits/Jamfile
+++ b/src/kits/Jamfile
@@ -71,6 +71,7 @@ SharedLibrary libbe_test.so :
 
 SubInclude HAIKU_TOP src kits app ;
 SubInclude HAIKU_TOP src kits bluetooth ;
+SubInclude HAIKU_TOP src kits contact ;
 SubInclude HAIKU_TOP src kits debug ;
 SubInclude HAIKU_TOP src kits device ;
 SubInclude HAIKU_TOP src kits game ;
